advanced_html_css/responsive_design/media_queries.md

```diff
@@ -43,18 +43,18 @@ You can also put any number of style definitions inside a media query:
 
-This does _not_ mean that you should just start your project with a media queries for each device. Each project is going to have different requirements based on the design you're trying to achieve. As mentioned above, try to limit your breakpoints to just what you _need_. With many relatively simple layouts, you can get by with only one mobile-centric breakpoint somewhere around `500`-`600px`. More complex layouts might benefit from doing a full sized layout above `1200px`, an altered "tablet" layout between `600px` and `1200px` and mobile below `600px`. The real takeaway here is that it doesn't really matter exactly where you set your breakpoints... just do what makes sense for your project.
+This does _not_ mean that you should just start your project with media queries for each device. Each project is going to have different requirements based on the design you're trying to achieve. As mentioned above, try to limit your breakpoints to just what you _need_. With many relatively simple layouts, you can get by with only one mobile-centric breakpoint somewhere around `500`-`600px`. More complex layouts might benefit from doing a full-sized layout above `1200px`, an altered "tablet" layout between `600px` and `1200px` and mobile below `600px`. The real takeaway here is that it doesn't really matter exactly where you set your breakpoints, just do what makes sense for your project.
 
 #### Zooming!
-Something to be aware of: in most browsers, zooming in on a webpage will change the effective resolution of that page. So if your browser window is exactly `1000px` wide, zooming in will cause the page to behave as if the screen is _smaller_, and will trigger media-queries based on the simulated/zoomed screen resolution. Zooming _out_ can be handy for debugging issues that arise on screens that are larger than your actual computer screen... and forgetting that you've zoomed in or out on a webpage can cause some real confusion when breakpoints refuse to trigger at the correct points.
+In most browsers, **zooming in on a webpage will change the effective resolution of that page**. If your browser window is exactly `1000px` wide, zooming in will cause the page to behave as if the screen is _smaller_, and will trigger media queries based on the simulated/zoomed screen resolution. Zooming _out_ can be handy for debugging issues that arise on screens that are larger than your own computer screen. Forgetting that you've zoomed in or out on a webpage can cause some real confusion when breakpoints refuse to trigger at the correct points.
```

Comment:
Nit: not sure we really need to bold this text.

Reply:
I bolded it because I removed the phrase 'Something to be aware of' which indicates that learners trip up on this issue often. Feel free to remove it.